docs/source/storeconv.md

```diff
@@ -176,20 +186,20 @@ range. That is:
    corresponding to those members only MUST be returned. In the case of
    `variables`, this will result in a dataset containing only the requested
    variables.
- - As a special case of the above, if an empty list is supplied, a dataset
+ - A special case of the above: If an empty list is supplied, a dataset
    containing *no data* MUST be returned -- but with the requested spatial and
    temporal dimensions.
 
 ### <a id="sec-datespec"></a>Date, time, and duration specifications
 
-In the common parameter `time_range`, times can be specified using the
+Within the common parameter `time_range`, times can be specified using the
```

Comment:
I'm not sure that this makes things clearer. On consideration, my "can" is also rather vague for a specification document, giving the impression that it's optional. I suggest rewriting this sentence as:

 - In values for the parameter `time_range`, the string components MUST conform to either the `date-time` or `time` JSON schema format.

Reply:
Stay with original.